spec/features/publish/courses/editing_degree_requirements_spec.rb

```diff
@@ -131,7 +131,7 @@
         end
 
         expect(publish_degree_subject_requirement_page.yes_radio).to be_checked
-        expect(publish_degree_subject_requirement_page.no_radio).not_to be_checked
+        expect(publish_degree_subject_requirement_page.the_no_radio).not_to be_checked
```

Comment:
Should have `the_yes_radio` as well, but I'd rather do `radio_yes` and `radio_no`.